File: Android/DayCareTeacher/DayCareTeacher/app/src/main/java/com/daycare/daycareteacher/model/DiaperModel.java
package com.daycare.daycareteacher.model;

import com.google.gson.annotations.SerializedName;

public class DiaperModel {
    @SerializedName("totalPages")
    public Integer totalPages;
    @SerializedName("totalRows")
    public Integer totalRows;
    @SerializedName("pageSize")
    public Integer pageSize;
    @SerializedName("id")
    public Integer id;
    @SerializedName("isExist")
    public Boolean isExist;
    @SerializedName("saveId")
    public Integer saveId;
    @SerializedName("statusCode")
    public Integer statusCode;
    @SerializedName("isSuccess")
    public Boolean isSuccess;
    @SerializedName("message")
    public String message;
    @SerializedName("data")
    public Data data;
    public class Data {

        @SerializedName("id")
        public Integer id;
        @SerializedName("agencyID")
        public Integer agencyID;
        @SerializedName("studentActivitiesID")
        public Integer studentActivitiesID;
        @SerializedName("diaperChangeTime")
        public String diaperChangeTime;
        @SerializedName("studentActivityDiaperNote")
        public String studentActivityDiaperNote;
        @SerializedName("activityTypeID")
        public Integer activityTypeID;
        @SerializedName("stringId")
        public Integer stringId;
        @SerializedName("studentID")
        public Integer studentID;
        @SerializedName("isActive")
        public Boolean isActive;
        @SerializedName("isDeleted")
        public Boolean isDeleted;
        @SerializedName("deletedBy")
        public Integer deletedBy;}
}

File: Android/DaycareParent/DaycareParent/app/src/main/java/com/daycare/daycareparent/model/DiaperModel.kt
package com.daycare.daycareparent.model

import com.google.gson.annotations.SerializedName

class DiaperModel {
    @SerializedName("totalPages")
    var totalPages: Int? = null
    @SerializedName("totalRows")
    var totalRows: Int? = null
    @SerializedName("pageSize")
    var pageSize: Int? = null
    @SerializedName("id")
    var id: Int? = null
    @SerializedName("isExist")
    var isExist: Boolean? = null
    @SerializedName("saveId")
    var saveId: Int? = null
    @SerializedName("statusCode")
    var statusCode: Int? = null
    @SerializedName("isSuccess")
    var isSuccess: Boolean? = null
    @SerializedName("message")
    var message: String? = null
    @SerializedName("data")
    var data: Data? = null

    inner class Data {

        @SerializedName("id")
        var id: Int? = null
        @SerializedName("agencyID")
        var agencyID: Int? = null
        @SerializedName("studentActivitiesID")
        var studentActivitiesID: Int? = null
        @SerializedName("diaperChangeTime")
        var diaperChangeTime: String? = null
        @SerializedName("studentActivityDiaperNote")
        var studentActivityDiaperNote: String? = null
        @SerializedName("activityTypeID")
        var activityTypeID: Int? = null
        @SerializedName("stringId")
        var stringId: Int? = null
        @SerializedName("studentID")
        var studentID: Int? = null
        @SerializedName("isActive")
        var isActive: Boolean? = null
        @SerializedName("isDeleted")
        var isDeleted: Boolean? = null
        @SerializedName("deletedBy")
        var deletedBy: Int? = null
    }
}
